_product/ce/qualcomm-robotics-rb5/README.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Qualcomm® Robotics RB5 Development Platform
```

Comment:
Maybe the front matter parse doesn't like the copyright character? Try the HTML entity for this character. `&#169;` If that doesn't work, try wrapping the string in quotes.

Reply:
its the same in the RB3 page, so i don't think that's it. could the CI not be deleting old assets from this commit?
one of the updates to this commit fixed some odd character formatting https://github.com/96boards/website/pull/1366/commits/12148c0ccfda36a52c0f00c23f6c5564de445cf0

Reply:
@ric96 - if you think that is the case, it may well be, we will have to wait until @pcolmer is back as he knows a lot more about how the GitHub actions CI job runner is setup to build this site.

Reply:
i think i got it, i wish yaml was more verbose about syntax indentation. had to go through line by line on  yamllint